### PR TITLE
librewolf browser support

### PR DESCRIPTION
**Pull request by TrueBob99**
Adds support for librewolf, a popular fork of firefox

### CODE REVIEW

**Comment by itsHanibee** [comment]
support for new browsers need to go in contrib/

**Comment by papadakis-k** [comment]
Hello.
The commits 01bd51ae31436abda75663e41ca029359fd5bd51 and 989754c270a38cb35a1d1803b98e0b088a50081b (on your repo @TrueBob99 ) revert part of your previous changes, resulting in no changes in `README.md` and `psd.conf`.
If I may help with what I understand you are trying to do, I would do something like this:
```
git rebase --interactive cd8c2a37f152bd2bde167a0e066085ac23bc17d9 patch-1
```
(cd8c2a37f152bd2bde167a0e066085ac23bc17d9 is where the master is now in this repo)
and edit the buffer to look like this
```
reword 1015acd Create librewolf
fixup 91a2500 Update psd.conf
fixup 3a2f3aa Update README.md
fixup bf72872 Update librewolf
# pick 989754c Update README.md
# pick 01bd51a Update psd.conf
fixup b1744fa Move to contrib
```
i.e. skip those two reverting commits, fixup everything into a single commit and maybe reword the single commit into something like "add LibreWolf support"

Of course you would need to force push the branch after (and the PR I believe autoupdates).
If all that sounds too complicated, please let me know and I can reproduce the PR on my account.
Kind regards.

**Comment by pern0x** [comment]
Hello,
could you please add the support for librewolf?
